layers/+lang/python/packages.el

```diff
@@ -257,7 +257,10 @@
 
       (defun python-setup-shell ()
         (if (executable-find "ipython")
-            (setq python-shell-interpreter "ipython")
+            (progn (setq python-shell-interpreter "ipython")
+                   (when (string-equal
+                          (nth 1 (split-string (shell-command-to-string "ipython --version") "")) "5"))
```

Comment:
May be for making it future-proof? 
`(version<= "5" (replace-regexp-in-string "\n$" ""  (shell-command-to-string "ipython --version")))`